src/context/AppProvider/AppContext.tsx

```diff
@@ -247,9 +263,16 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
     // Exclude assets for which we are unable to get market data
     // We would fire query thunks / XHRs that would slow down the app
     // We insert price data for these as resolver-level, and are unable to get market data
-    const excluded: AssetId[] = uniV2LpIdsData.data ?? []
+    const excluded: AssetId[] = nftAssetIds.concat(uniV2LpIdsData.data ?? [])
```

Comment:
as discussed we want to do this concept of blacklist/unsupported on the provider side (market service manager), and not the consumer side (app context)

we could add state to the market service manager, such that other services can let it know that it's unable to get market data, e.g.

```
const uniV2LPIds = [...]

marketServiceManager.markUnsupportedAssetIds(uniV2LPIds)
```

this way, the market service is unaware of univ2 logic, and consumers (this effect) don't have to be aware of what they can or cannot ask for

Reply:
https://github.com/shapeshift/web/pull/4555/commits/ef6955e97c7f8d78efe74cbd05d03d69edae633f will save univ2 and further abstraction for later as not related to this pr directly